metrics/test_driver/cli/selenium/driver/firefox_driver.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:io';
+
+import 'package:archive/archive.dart';
+
+import '../../../common/config/driver_tests_config.dart';
+import '../../../util/file_utils.dart';
+
+/// Represents the firefox driver.
+class FirefoxDriver {
```

Comment:
Same as the ChromeDriver does nothing if a file exists - not sure is it right.

Reply:
The ChromeDriver and FirefoxDriver classes are needed to download the driver files if they do not exist. We should not run them manually. The selenium server will run them if required. So, yeah, we should do nothing if these files exist.

Reply:
Oh, thanks! Didn't know that)